test_flask.py

```
@@ -0,0 +1,67 @@
+    with app.app_context():
+        print("✅ Flask app created successfully")
+
+        # Test static file access
+        from flask import url_for
+        try:
+            css_url = url_for('static', filename='css/styles.css')
+            print(f"✅ CSS URL generated: {css_url}")
+
+            js_url = url_for('static', filename='js/theme-manager.js')
+            print(f"✅ JS URL generated: {js_url}")
+        except Exception as e:
```

Ruff (0.13.1)

Do not catch blind exception: `Exception`

(BLE001)